README.md

```diff
@@ -437,6 +437,14 @@ Like the `useEffect` family of hooks, this hook doesn't add any new parameters t
   useEffectStreamResourceWhenDepsReadyBy[D](deps: Ctx => Pot[D])(stream: Ctx => D => Resource[IO, fs2.Stream[IO, Unit]])
 ```
 
+### `useSignalStream` / `useSignalStreamByReuse`
+
+Given a value, creates an `fs2.Stream` that will emit a new value every time the value changes. Equality is tested by `Eq`/`Reusability`. The stream is created when the component is mounted and memoized. This is backed by a `SignallingRef`, so intermediate values may be dropped during rapid bursts.
+
+``` scala
+  useValueStream[A: Eq](value: A): Reusable[Pot[fs2.Stream[DefaultA, A]]]
```

Comment:
Late to the party, but I think this should be `useSignalStream`?

Reply:
Yup, good catch, it was a last-minute rename.